### DISCUSSION TITLE
How can i write additional text over note?

### DISCUSSION THREAD

**Discussion by metaIhead** [Perlence/PyGuitarPro#30] [Q&A]
![изображение](https://user-images.githubusercontent.com/36397154/185082316-710c4dbf-a312-43c7-a390-e6e89b6b980c.png)

**Comment by KyleAbent** [comment]
It's a beat text ![IMG_20220814_021712_596.jpg](https://user-images.githubusercontent.com/12185724/185107233-0ccb1278-3dc7-4ffe-8157-6a1e432e347f.jpg)
